pkg/cmd/gtctl/cluster/list/list.go

```diff
@@ -30,24 +32,29 @@ func NewListClustersCommand(l logger.Logger) *cobra.Command {
 		Short: "List all GreptimeDB clusters",
 		Long:  `List all GreptimeDB clusters`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			m, err := manager.New(l, false)
+			k8sDeployer, err := k8s.NewDeployer(l)
 			if err != nil {
 				return err
 			}
 
 			ctx := context.TODO()
-			clusters, err := m.ListClusters(ctx, &manager.ListClusterOptions{})
+			clusters, err := k8sDeployer.ListGreptimeDBClusters(ctx, nil)
 			if err != nil && !errors.IsNotFound(err) {
 				return err
 			}
-			if errors.IsNotFound(err) || (clusters != nil && len(clusters.Items) == 0) {
+			if errors.IsNotFound(err) || (clusters != nil && len(clusters) == 0) {
```

Comment:
Maybe is `if errors.IsNotFound(err) || (clusters != nil && len(clusters.Item) == 0)` ?

Reply:
Here can:
```
if errors.IsNotFound(err) || (clusters != nil && len(clusters) == 0) {
  l.Error("clusters not found\n")
  return nil
}

clusters, err := k8sDeployer.ListGreptimeDBClusters(ctx, nil)
	if err != nil  { // Delete check [!errors.IsNotFound(err)].
        return err
}
```

Reply:
As the above comment says, you can provide another PR to handle that.